ngraph/frontend/onnx_import/src/op/roi_align.cpp

```diff
@@ -39,8 +39,8 @@ namespace ngraph
                     const auto& rois = inputs[1];
                     const auto& num_rois = inputs[2];
 
-                    const auto pooled_h = node.get_attribute_value<int64_t>("output_height", 1);
-                    const auto pooled_w = node.get_attribute_value<int64_t>("output_width", 1);
+                    const auto pooled_h = node.get_attribute_value<int64_t>("pooled_h", 1);
+                    const auto pooled_w = node.get_attribute_value<int64_t>("pooled_w", 1);
```

Comment:
Why did you change the attribute names? Is it a bug which we have before?

Reply:
Documentation specifies the attribute names to be pooled_h and pooled_w: [here](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/detection/ROIAlign_3.md)

Reply:
This is a good catch - this code retrieves the attributes from an ONNX node which means you should follow this spec https://github.com/onnx/onnx/blob/master/docs/Operators.md#RoiAlign

Reply:
Ok, looks like we should revert this change.

Reply:
I`ve reverted it